src/components/organisms/pupil/browse/OakPupilJourneySubjectButton/__snapshots__/OakPupilJourneySubjectButton.test.tsx.snap

```diff
@@ -157,9 +159,12 @@ exports[`OakPupilJourneySubjectButton matches snapshot 1`] = `
   />
   <button
     className="c3 c4 internal-button"
+    font="heading-7"
     onClick={[Function]}
     onMouseEnter={[Function]}
     onMouseLeave={[Function]}
+    ph="inner-padding-s"
+    pv={null}
   >
```

Comment:
I am unfamiliar with this codebase and these files appear to have crept in after running test:snapshot